.github/workflows/installer.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Installer
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'packaging/installer/install-required-packages.sh'
+  pull_request:
+    paths:
+      - 'packaging/installer/install-required-packages.sh'
+jobs:
+  install-required-packages:
+    name: Required Packages
+    strategy:
+      matrix:
+        distro:
+          - 'alpine:3.11'
+          - 'alpine:3.10'
+          - 'alpine:3.9'
+          - 'archlinux:20200106'
+          - 'archlinux:20191205'
+          - 'archlinux:20191105'
+          - 'centos:8'
+          - 'centos:7'
+          - 'centos:6'
+          - 'debian:buster'
+          - 'debian:jessie'
+          - 'debian:stretch'
+          - 'fedora:31'
+          - 'fedora:30'
+          - 'fedora:29'
+          - 'opensuse/leap:15.2'
+          - 'opensuse/leap:15.1'
+          - 'opensuse/leap:15.0'
+          - 'opensuse/tumbleweed:latest'
+          - 'ubuntu:20.04'
+          - 'ubuntu:19.10'
+          - 'ubuntu:18.04'
```

Comment:
Unless something has changed without mee knowing about it, we still support 16.04 and 19.04, and should be testing on those as well.

Reply:
Done